src/Masa.Blazor/Components/Switch/MSwitch.cs

```diff
@@ -62,6 +62,16 @@ public override string ValidationState
 
         public string TextColor => HasText ? ComputedColor : (IsLoading ? null : ValidationState);
```

Comment:
'MSwitch.TextColor' hides inherited member 'MInput<bool>.TextColor'. Use the new keyword if hiding was intended.